.travis.yml

```diff
@@ -61,6 +61,34 @@ matrix:
           packages:
             - gcc-multilib
             - libgmp-dev:i386
+
+    # clang with memory sanitizer
+    #
+    # --disable-openssl-tests because openssl uses uninitialized memory. ASM
```

Comment:
nit: I haven't checked this myself, but an alternative might be to `export MSAN_OPTIONS` with a suppressions file for the openssl function that is affected.

Reply:
Indeed but I don't think that's better in the end, and disabling the tests as done here is simpler. Note that the flag here really just influences just the test code. OpenSSL is only used in the tests, not used in the library itself.